scss/_button-group.scss

```diff
@@ -34,6 +34,9 @@
 }
 
 .btn-group {
+  // Prevent white corners if shadow or shadow-lg property applied
```

Comment:
This comment isn't really needed, the property and value speak for themselves. 

Then I guess it'd be good to go 👌